tests/tests/operation_tests.cpp

```diff
@@ -5712,6 +5712,58 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
         FC_LOG_AND_RETHROW()
     }
 
+    BOOST_AUTO_TEST_CASE(transfer_from_savings_when_memo_storing_specified) {
+         try {
+            BOOST_TEST_MESSAGE("Testing: transfer_from_savings_when_memo_storing_specified");
+
+            ACTORS((alice)(bob));
+            generate_block();
+
+            fund("alice", ASSET("10.000 GOLOS"));
+
+            transfer_to_savings_operation save;
+            save.from = "alice";
+            save.to = "alice";
+            save.amount = ASSET("10.000 GOLOS");
+
+            signed_transaction tx;
+            tx.operations.push_back(save);
+            tx.set_expiration(db->head_block_time() + STEEMIT_MAX_TIME_UNTIL_EXPIRATION);
+            tx.sign(alice_private_key, db->get_chain_id());
+            db->push_transaction(tx, 0);
```

Comment:
@afalaleev Just transfer_from_savings_apply without any changes is that.

Reply:
@afalaleev This test is semi-useless because plugin sets it anyway https://github.com/GolosChain/golos/blob/797-config-storing-memo-in-transfer/plugins/chain/plugin.cpp#L349
But I'll add.

Reply:
By default, the chain-plugin set flag to `true` and you should test it.

Reply:
@afalaleev  It sets flag to true with calling `db.set_store_memo_in_savings_withdraws`.